examples/prometheus/prometheus.yaml

```diff
@@ -72,7 +73,7 @@ objects:
           imagePullPolicy: IfNotPresent
           name: prometheus
           ports:
-          - containerPort: 9090
```

Comment:
The process inside the pod still listens on 9090. There's likely something you need to change in the config to have it listen on 8080.